topic04-git-introduction/book-git-tutorial/02.Branching.md

```diff
@@ -0,0 +1,40 @@
+# Git Branching
+
+----
+## What is Branching?
+
+
+This document is to show how branching works in git and how to use it. Branching is a feature available in git that lets you pull off from the master branch to make changes that won't directly effect the master branch unless merged. It also means that others have to review your work first just in case you have errors in your work such as spelling or bugs.
+
+## Why Would You Use Branching
+
+Branching is better than VCS in alot of different ways and one of them is disk space. Branching is all online and you don't have to keep track of your different versions of software which in turn will save space on your hard drive/ssd. 
```

Comment:
Not sure about this. Don't know if it relates to how we are using branching. We are talking about local changes so the saving space comment is a bit misleading